src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -2091,11 +2114,23 @@ char *os::scan_pages(char *start, char* end, page_info* page_expected, page_info
 }
 
 
-bool os::pd_uncommit_memory(char* addr, size_t size) {
-#ifdef __OpenBSD__
+MACOS_ONLY(bool os::pd_uncommit_memory(char* addr, size_t size, bool executable))
+NOT_MACOS(bool os::pd_uncommit_memory(char* addr, size_t size)) {
```

Comment:
Out of curiosity, did you try and build BSD? Does anyone? Or does it just quietly bitrot away?

At some point maybe the delta between BSD and MacOS are large enough to split the bases (and maybe retire BSD unless someone maintains it)

Reply:
Not me, but some may build it for free/open bsd